### PR TITLE
Fix checking /_app

### PR DESCRIPTION
**Pull request by TheMatrixan**
Fixes: #35 

[ ] Documentation update
[+] Bug fix
[ ] New feature
[ ] Other, please explain:

**What changes did you make? (Give an overview)**

**Which issue (if any) does this pull request address?**

**Is there anything you'd like reviewers to focus on?**

### CODE REVIEW

File: src/loader.ts [hunk 1]
```diff
@@ -116,14 +116,14 @@ export default function loader(
     //
     // This way, the only modified file has to be the _app.js.
     if (hasGetInitialPropsOnAppJs) {
-      return pageNoExt === '/_app'
+      return pageNoExt.startsWith('/_app')
```
Comment (aralroca) [inline src/loader.ts:119]: the problem is that this is not going to work in monorepos...

Comment (aralroca) [reply src/loader.ts:119]: well, maybe yes is fine. If before was just `===` and was working...  Is `pageNoExt` only the page and not all the path... So then yes.  This is to fix these cases `/_app.page.ts`, right?

**Review by aralroca — COMMENTED** [review]
Thanks for the contribution! LGTM. The only thing... can you add a test similar than this?

https://github.com/aralroca/next-translate-plugin/blob/60c4ee7d520ad9df789848ea742280d2b8befd49/__tests__/loader.test.js#L66-L79

Thanks!!

File: src/loader.ts (hunk 1 above)
Comment (TheMatrixan) [reply src/loader.ts:119]: Yes, this is a fix for `_app.page.tsx` - custom `pageExtensions`.

**Comment by TheMatrixan** [comment]
Okay, done, @aralroca

**Review by aralroca — APPROVED** [review]

**Comment by aralroca** [comment]
@TheMatrixan thanks for your contribution. Your code is already available in [2.5.3-canary.2)](https://github.com/aralroca/next-translate-plugin/releases/tag/2.5.3-canary.2) prerelase

**Pull request merged**